docs/platforms/python/tracing/instrumentation/performance-metrics.mdx

```diff
@@ -1,27 +0,0 @@
----
-title: Performance Metrics
```

Comment:
Is this being moved under span metrics?

Reply:
In the JavaScript docs, we were asked to keep this one separate out.